docs/architecture-decisions/0007-use-devise-for-authentication.md

```diff
@@ -0,0 +1,104 @@
+# 7. Use Devise for authentication
+
+Date: 2024-07-11
+
+## Status
+
+Accepted
+
+## Context
+
+We will need to authenticate users in TACOS to help manage access to the staff interface. The ideal authentication
+approach would involve Omniauth integration, as IS&T recently implemented Okta, and
+[a strategy already exists](https://github.com/omniauth/omniauth-okta) to authenticate in Okta using Omniauth.
```

Comment:
I don't think we should use that omniauth integration as it seems unsupported, https://github.com/omniauth/omniauth_openid_connect is more likely the path forward.